src/raftstore/store/worker/split_check.rs

```diff
@@ -227,15 +227,37 @@ impl<C: Sender<Msg>> Runner<C> {
                 let res = host.approximate_split_keys(region, &self.engine);
                 if let Err(e) = res {
                     error!(
-                        "[region {}] failed to get approxiamte split key: {}",
+                        "[region {}] failed to get approximate split key: {}, try scan way",
                         region_id, e
                     );
-                    return;
+                    let timer = CHECK_SPILT_HISTOGRAM.start_coarse_timer();
```

Comment:
duplicated code with the above scan policy.

Reply:
yeah, it is just a quick fix to build a patch. I will change it.